### ISSUE TITLE
See Experimental result by see the train validation loss curve

### ISSUE THREAD

**Issue by ethio-artifical** [hulianyuyy/CorrNet#56]
Hello, i want to examine the model by seeing the train and validation loss but i can not get the validation loss how can i get and plot the graph?? can you help me Please

**Comment by hulianyuyy** [comment]
Currently, it doesn't calculate the val loss in the validation procedure. To achieve this, you can conduct the ```model.criterion_calculation``` function in the ```seq_eval``` function in the seq_scripts.py and return the loss values, like what the ```seq_train``` function in the seq_scripts.py does.

**Comment by ethio-artifical** [comment]
Thank you it works perefectly
